common/src/main/java/com/wynntils/models/npcdialogue/NpcDialogueModel.java

```diff
@@ -31,21 +30,16 @@ public class NpcDialogueModel extends Model {
     private final Set<Feature> dialogExtractionDependents = new HashSet<>();
 
     private NpcDialogue currentDialogue = NpcDialogue.EMPTY;
-    private final List<ConfirmationlessDialogue> confirmationlessDialogues = new ArrayList<>();
+    private ConcurrentLinkedQueue<NpcDialogue> confirmationlessDialogues = new ConcurrentLinkedQueue<>();
```

Comment:
Why ConcurrentLinkedQueue? In what way can this be concurrently modified?

I'm not questioning this change per se, I am just curious how this happens. (And maybe asking that the access pattern should be documented so we understand any race situations.)

Reply:
If we translate a confirmationless dialogue, then we need to change the list. However, translation is async.

I will make sure to comment this.